### ISSUE TITLE
kill-line kills all subsequent lines

### ISSUE THREAD

**Issue by JeffreyBenjaminBrown** [joshsh/brain-mode#27]
Running `kill-line` (`C-k`) and then pushing the view causes everything after that note to disappear too.

Fortunately, highlighting the note and then running `kill-region` (`C-w`) still behaves.

**Comment by JeffreyBenjaminBrown** [comment]
Bah! You warned me about this. It's the "don't leave a blank line in a pushed view" problem.

**Comment by joshsh** [comment]
Yes, we knew this was going to be an issue. However, since it is so disruptive, I don't mind keeping this issue open until the next format bump resolves it.
